### ISSUE TITLE
Refactor PaymentProvisioning component to use core component structure - Part 1

### ISSUE THREAD

**Issue by colganmeanor** [justifi-tech/web-component-library#677]
Refactor PaymentProvisioning component to use core component structure. This issue / task represents the working branch (Payment-provisioning-component-form-steps-render-refactor) that refactors the first five form steps of PaymentProvisioning to use the core component structure we've begun implementing on our other components. 

The PRs that were merged into the working branch are:

- [Core Info](https://github.com/justifi-tech/web-component-library/pull/606)
- [Legal Address](https://github.com/justifi-tech/web-component-library/pull/612)
- [Additional Questions ](https://github.com/justifi-tech/web-component-library/pull/613)
- [Representative](https://github.com/justifi-tech/web-component-library/pull/648)
- [Owners](https://github.com/justifi-tech/web-component-library/pull/646)

Each of these PRs has already been approved and QA'd - so ideally it should not take too long to review the working branch before merging into `main`.

Once this is merged - I will start working on the remaining form components as time allows.
